components/raftstore/src/store/peer.rs

```diff
@@ -3786,19 +3891,22 @@ where
             return;
         }
 
-        // Reverse sort peers based on `next_idx`, then try to get a potential quorum.
-        next_idxs.sort_by(|x, y| y.1.cmp(&x.1));
         let (mut potential_quorum, mut quorum_ok) = (HashSet::default(), false);
-        for &(peer_id, _, usage) in &next_idxs {
-            if matches!(usage, Some(DiskUsage::AlreadyFull)) {
-                continue;
-            }
```

Comment:
If a peer's disk is full but it is in `potential_quorum` we still set its `inflight_msgs` to 1?

Reply:
As comment, we need support CC even during the 2nd level disk usage.